frontend/src/components/PLAYGROUND/ReportSelector/handleExport.tsx

```diff
@@ -0,0 +1,49 @@
+import { ConvertToStringItem, UpdatedSearchParams } from "types";
+import { apiAxios } from "utils";
+
+const convertValueToString = (item: ConvertToStringItem) => {
```

Comment:
this seems like a function that would be better served in a utils file/folder

Reply:
moved to utils folder, good suggestion